packages/vue-app/template/components/nuxt-link.client.js

```diff
@@ -0,0 +1,85 @@
+<%= isTest ? '// @vue/component' : '' %>
+import Vue from 'vue'
+
+const requestIdleCallback = window.requestIdleCallback ||
```

Comment:
Do you have an example?

Reply:
Because it's just a fake polyfill of `window.requestIdleCallback`

Reply:
No just suggesting to replace it with something like `window.NuxtrequestIdleCallback` indicating it is a nuxt related callback :D

Reply:
Since it's a pretty poor polyfill and it's used only in this particular area, I suggest to not add this function to the globals and keep it local (at the moment, and could be moved to the utils)